test/regression/sql/type_support.sql

```diff
@@ -131,10 +148,10 @@ SELECT * FROM uuid_tbl;
 -- JSON
 CREATE TABLE json_tbl(a JSON);
 INSERT INTO json_tbl SELECT CAST(a as JSON) FROM (VALUES
-    ('{"key1": "value1", "key2": "value2"}'),
-    ('["item1", "item2", "item3"]'),
-    (NULL),
-    ('{}')
+	('{"key1": "value1", "key2": "value2"}'),
+	('["item1", "item2", "item3"]'),
+	(NULL),
+	('{}')
```

Comment:
nit: according to our `.editorconfig` we use spaces for indentation in our SQL files. (same with the indentation above)

Reply:
Converted whole sql file to space indentation.